rust/libnewsboat/src/utils.rs

```diff
@@ -265,8 +267,78 @@ pub fn unescape_url(rs_str: String) -> Option<String> {
     Some(result.unwrap().replace("\0",""))
 }
 
+pub fn get_command_output(cmd: &str) -> String {
+    let cmd = Command::new("sh").arg("-c").arg(cmd).output();
+    // from_utf8_lossy will convert any bad bytes to U+FFFD
+    cmd.map(|cmd| String::from_utf8_lossy(&cmd.stdout).into_owned())
+        .unwrap_or_else(|_| String::from(""))
+}
+
+pub fn run_command(cmd: &str, param: &str) {
+    let child = Command::new(cmd)
+        .arg(param)
+        .stdin(Stdio::null())
+        .stdout(Stdio::null())
+        .stderr(Stdio::null())
```

Comment:
I guess setting it explicitly to `/dev/null` is necesarry because otherwise the output will end up in the terminal window where newsboat is running?

Reply:
Yes.

Okay, okay, fine, I'll add the comment :)